src/frontend/App/context/BehandlingContext.tsx

```diff
@@ -83,6 +84,8 @@ const [BehandlingProvider, useBehandling] = constate(() => {
         // eslint-disable-next-line
     }, [behandling]);
 
+    const useHentVilkårState = useHentVilkår();
```

Comment:
Er dette greit? Eller burde det løses på en annen måte?

Reply:
Bør vi implementere callback og "rerun" på denne, slik som på useBehandling?
Da har vi iallefall mulighet til å "splitte" oppdaterPersonopplysninger ut av useHentVilkår. For oppdaterPersonopplysninger bør jo egentlig hente både vilkår og behandling på nytt - dersom vi ser for oss å tulle med vilkårene (da må jo tilstanden til behandlingen resettes til utredes også)

Reply:
Ja, mulig :) 
Strengt tatt trenger man aldri å kalle hentVilkår med en behandlingId hvis man gjør det sånn, som er ryddigere
Tenker du på "oppdaterGrunnlagsdataOgHentVilkår"? (oppdaterPersonopplysninger)